api/client/workflows_v1_alpha.go

```diff
@@ -38,3 +40,31 @@ func (c *WorkflowApiV1AlphaApi) ListWorkflows(project_id string) (*models.Workfl
 
 	return models.NewWorkflowListV1AlphaFromJson(body)
 }
+
+func (c *WorkflowApiV1AlphaApi) CreateWf(project_id string) ([]byte, error) {
+	requestToken, err := uuid.NewUUID()
+
+	req, err := models.NewWorkflowV1AlphaSnapshotRequest(project_id, "archive content", requestToken.String())
+
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("creating reuild request failed '%s'", err))
+	}
+
+	requestBody, _ := json.Marshal(req)
+
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("request body creation failed '%s'", err))
+	}
+
+	body, status, err := c.BaseClient.Post(c.ResourceNamePlural, requestBody)
+
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("connecting to Semaphore failed '%s'", err))
+	}
+
+	if status != 200 {
+		return nil, errors.New(fmt.Sprintf("http status %d with message \"%s\" received from upstream", status, body))
```

Comment:
This `error.New(fmt.Sprintf(` is probably coming from the time when I didn't know that there is `fmt.Errorf("error %s %s %s", a, b, c)` is available and shorter.

Reply:
thank you